packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -177,6 +177,13 @@ plan.addApiStage({
 });
 ```
 
+The value of the API Key will be automatically generated by API Gateway.
+Alternatively, the API Key value can be specified during key creation.
+
+```ts
+const key = api.addApiKey('ApiKey', 'MyApiKeyThatIsAtLeast20Characters');
+```
```

Comment:
Documentation is not aligned with code.

Reply:
D'oh. Whoops! Posted an update to the README.